pkg/lib/template/template_loader.go

```diff
@@ -303,6 +304,34 @@ func (loader *Loader) Render(
 		"server_url": func() *url.URL {
 			return loader.ServerURL
 		},
+		"int_range": func(s, e interface{}) []int64 {
+			sv := reflect.ValueOf(s)
+			ev := reflect.ValueOf(e)
+			var start, end int64
+			switch sv.Kind() {
+			case reflect.Int:
+				start = int64(s.(int))
+			case reflect.Int64:
+				start = s.(int64)
+			default:
+				return nil
```

Comment:
I order to switch types (why do you btw? template literals should always be `int`), you want to use a Go type switch. There is no need for reflect and type switching is faster and more elegant.

```go
var from, to int64
switch v := (s).type {
  case int64:
     from = v
  case int:
    from = int64(s)
  default:
   panic()
}

switch v := (e).type {
  case int64:
    to = v
  case int:
    to = int64(s)
  default:
   panic()
}

```